Gemfile

```diff
@@ -5,6 +5,9 @@ gem "better_errors"
 gem 'aws-sdk'
 gem 'annotate'
 
+# Search functionality
+gem 'searchkick'
```

Comment:
@nikitr (nit) don't include things that aren't relevant for your feature. you should create a new branch for the elastic search + searchkick installment!